source/common/event/file_event_impl.h

```diff
@@ -41,5 +45,119 @@ class FileEventImpl : public FileEvent, ImplBase {
   // polling and activating new fd events.
   const bool activate_fd_events_next_event_loop_;
 };
+
+// Forward declare for friend class.
+class UserSpaceFileEventFactory;
+
+// The interface of populating event watcher and obtaining the active events. The events includes
+// Read, Write and Closed.
```

Comment:
includes -> include

Reply:
I think instead of listing events you can refer to FileReadyType

Reply:
:+1: